docs/start/envubuntu.md

```diff
@@ -11,7 +11,7 @@ Tested on 16.04 LTS (Yosemite) and 14.04 LTS (El Capitan).  Not domain joined.
 ## Dependency Packages
 
 ```bash
-sudo apt-get install -y libunwind8 libcurl3
+sudo apt-get install -y libunwind8 libcurl3 libicu52
```

Comment:
will this be fine on Ubuntu 16 or should we break apart and specify a list for 14 and one for 16?